spec/models/element_aviti_sequencing_pipeline_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe ElementAvitiSequencingPipeline, type: :model do
+  describe '#post_release_batch' do
+    let(:pipeline) { create(:element_aviti_sequencing_pipeline) }
+    let(:batch) { create(:batch) }
+
+    it 'calls Messenger with eseq_flowcell root' do
+      allow(Messenger).to receive(:create!)
```

Comment:
This stubbing is OK for checking that the overridden method calls the Messenger with correct arguments but it would be good to have an additional test on the rendered flowcell message using the flowcell IO template. The superclass has a similar test. It will give us a chance to make assertions on the content.